File: 2025_03/[코딩 기초 트레이닝] 꼬리 문자열/꼬리 문자열.py
def solution(str_list, ex):
    answer = ''
    for str in str_list:
        if ex not in str:
            answer += str_list
    return 

# lambda 이용 풀이
def solution(str_list, ex):
    return ''.join(filter(lambda x: ex not in x, str_list))

File: 2025_03/[코딩 기초 트레이닝] 꼬리 문자열/꼬리 문자열.js
function solution(str_list, ex) {
    let result = '';

    for(let i = 0; i < str_list.length; i++) {
        if(!str_list[i].includes(ex)) {
            result += str_list[i];
        }
    }    

    return result;
}

// reduce 이용 풀이
function solution(str_list, ex) {
  return str_list.reduce((acc, cur) => {
    if(cur.includes(ex)) return acc;
    return acc+cur;
  }, "");
}